h/security/policy/helpers.py

```diff
@@ -0,0 +1,4 @@
+@staticmethod
```

Comment:
Is `@staticmethod` still needed here?

Reply:
Nope, well spotted. I'm surprised tests or linting didn't catch that :man_shrugging: